andy_api/api/api_routes.py

```diff
@@ -80,6 +81,12 @@ def get_response():
     """
     if request.method == "POST":
         session_id = request.args.get('session_id')
+        board_str = request.args.get('board_str')
+
+        # Update board_str stored in database
+        latest_board_str = shelve.open('./db/latest_board_str')
+        latest_board_str ['board_str'] = board_str
+        latest_board_str.close()
```

Comment:
Rather than storing this using `shelve`, can we just pass around `board_str` as a parameter for the functions for determining intent? We wouldn't have to close and open `shelve` a bunch of times.